README.md

```diff
@@ -1,6 +1,9 @@
 # M2-ISA-R v2
 
-This tool generates architecture models for ETISS from CoreDSL source files or abstract model trees.
+This tool generates architecture models for [ETISS](https://github.com/tum-ei-eda/etiss) from CoreDSL source files or abstract model trees. It is the 2nd version of the M2-ISA-R tool (not to be confused with the CoreDSL language versions, of which this tool is only compatible with version 1, see [below](##CoreDSL)).
```

Comment:
Technically this tool is only compatible with its own dialect of CoreDSL, unofficially called CoreDSL 1.5

Reply:
Ahh, okay. Should I include that in the readme? Are there any plans from your side to support CoreDSL v2 in the future?